parachains/runtimes/bridge-hubs/bridge-hub-kusama/src/bridge_hub_config.rs

```diff
@@ -47,6 +47,16 @@ parameter_types! {
 	pub PolkadotGlobalConsensusNetwork: NetworkId = NetworkId::Polkadot;
 	// see the `FEE_BOOST_PER_MESSAGE` constant to get the meaning of this value
 	pub PriorityBoostPerMessage: u64 = 91_022_222_222_222;
+
+	pub AssetHubKusamaParaId: cumulus_primitives_core::ParaId = 1000.into();
+
+	pub FromAssetHubKusamaToAssetHubPolkadotRoute: SenderAndLane = SenderAndLane::new(
+		ParentThen(X1(Parachain(AssetHubKusamaParaId::get().into()))).into(),
+		ASSET_HUB_KUSAMA_TO_ASSET_HUB_POLKADOT_LANE_ID,
+	);
+
+	pub CongestedMessage: Xcm<()> = unimplemented!("TODO: not supported yet!");
```

Comment:
yes, I was backporting from your branch and I left this as a last point, because I used different pallet instance (43) for router,
and I was thinking about something like we have for bridge calls: https://github.com/paritytech/parity-bridges-common/blob/master/relays/client-bridge-hub-kusama/src/runtime_wrapper.rs#L48

where do you think we should put AHK/AHP primitives?
in bridges repo or just somewhere in Cumulus repo?

Reply:
> I used different pallet instance (43) for router

Ah, ok then. I was thinking about our `bp-bridge-hub-kusama` and `bp-bridge-hub-polkadot`

Reply:
ahaaa, ok, got it, I thought first about `bp-asset-hub-*` in bridges :)
thank you, yeah, good idea,
I will put it `bp-bridge-hub-*` and also with that constant you suggested [here](https://github.com/paritytech/cumulus/pull/2997/files#r1294270498)

and e.g. then in tests for `asset-hub-kusama`  I will just add dev-dependency `bp-bridge-hub-kusama` like you suggested

Reply:
:facepalm: Sorry - I mean `bp-asset-hub-*` of course, because we'll be testing it in the asset hub runtime. Then we won't need to refer the bridge hubs from asset hubs. But the opposite (`bp-bridge-hub-*`) is also lgtm :)

Reply:
ok, fixed also with sanity checks for max weights